shell/platform/tizen/BUILD.gn

```diff
@@ -139,6 +139,20 @@ template("embedder_for_profile") {
       ]
     }
 
+    if (profile == "common") {
+      libs += [
+        "capi-base-common",
+        "feedback",
+      ]
+    }
+
+    if (profile == "tv") {
+      libs += [
+        "capi-base-common",
+        "feedback",
```

Comment:
It seems the same dependencies are applicable for all profiles so you may move these to common `libs`.